physical/mongo_test.go

```diff
@@ -0,0 +1,58 @@
+package physical
+
+import (
+	"os"
+	"testing"
+
+	"github.com/hashicorp/vault/helper/logformat"
+	log "github.com/mgutz/logxi/v1"
+)
+
+func TestMongoBackend(t *testing.T) {
+	logger := logformat.NewVaultLogger(log.LevelTrace)
+
+	mongoUri := os.Getenv("MONGODB_URI")
```

Comment:
We've been switching things over to `dockertest` (https://github.com/ory-am/dockertest); currently the Consul backend uses it (out of the physical backends; more of the secret backends use it). Please use that for testing so that tests *always* run. It's pretty easy to set up. You can see/steal some code from `builtin/logical/mongo/backend_test.go` as it already uses `dockertest`!